### PR TITLE
Update New features and enhancements on May 20 Release Notes

### PR DESCRIPTION
**Pull request by jmbermingham**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Enter a brief description of your changes here to communicate to the maintainers what you changed and why.

## Checklist for contributors

Before submitting this PR, please make sure:

* [ ] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [ ] You have signed the [Axway CLA](https://cla.axway.com/)
* [ ] You have verified the technical accuracy of your change
* [ ] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [ ] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [ ] You have verified that all status checks have passed

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

## Checklist for approvers

_This section is to be filled out by project maintainers only._

Before approving this PR, please make sure it:

* [ ] Passes all status checks
* [ ] Does not have conflicts with the base branch
* [ ] Is clear and complete
* [ ] Is believed to be accurate (technical accuracy to be checked with an SME for PRs originating from outside R&D)
* [ ] Follows [Axway style](https://techweb.axway.com/confluence/display/RIE/Style+guide)
* [ ] Follows [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)
* [ ] Follows [best practices](https://axway-open-docs.netlify.com/docs/contribution_guidelines/bestpracticedevdoc/)
* [ ] Does not contain typos, grammatical errors, etc.
* [ ] Does not expose any sensitive information
* [ ] Passes the Markdown linter rules (automated via CI check)
* [ ] Does not contain broken links
* [ ] Is discoverable and navigable

## Checklist for mergers

_This section is to be filled out by project maintainers only._

Before merging this PR, please make sure:

* [ ] You have applied a size label (unless this work is tracked in the ReadMeFirst JIRA project)
* [ ] You have applied a production label if this needs a manual push to Zoomin
* [ ] You have added it to the correct GitHub project
* [ ] You have added it to the correct Sprint milestone
* [ ] You have appended a `fixes` line to this description if this PR fixes a specific GitHub issue
* [ ] You have added a new collection to Netlify CMS config (static/admin/config.js) if required (new doc sections)
* [ ] You have updated the Zoomin classification file if required (new AMPC topics)

### CODE REVIEW

File: content/en/docs/apim_relnotes/20200530_apimgr_relnotes.md [hunk 1]
```diff
@@ -14,9 +14,21 @@ Docker deployment is supported on Linux. For a summary of the system requirement
 
 ## New features and enhancements
 
-The following new features and enhancements are available in this update.
+### User management module
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20200530_apimgr_relnotes.md:17]: @jmbermingham The information here is too generic. I don't see how this helps, or it's informative to customers. Can you please give me the link to this new module so I can link it from the text

File: content/en/docs/apim_relnotes/20200530_apimgr_relnotes.md [hunk 2]
```diff
@@ -14,9 +14,21 @@ Docker deployment is supported on Linux. For a summary of the system requirement
...
+
+The operating systems CentOS 7, CentOS 8, RedHat 7, and RedHat 8 have been tested and validated.
+
+### Improved upgrade experience
+
+* You can use the script `update_apigw.sh` for updating API Gateway and API Manager installations, replacing the post installation script `apigw_sp_post_install.sh`.
+* Automation of some manual steps for update.
+* The `update_apimanager.sh` script still required.
+* The update is still applied manually to each node in the topology.
+* There are new scripts to apply updates for Policy Studio and Configuration Studio.
```
Comment (andreamussap) [inline content/en/docs/apim_relnotes/20200530_apimgr_relnotes.md:31]: @jmbermingham 'There are new scripts' - can we link to this page? https://axway-open-docs.netlify.app/docs/apim_reference/scripts_changelog_sp/

File: content/en/docs/apim_relnotes/20200530_apimgr_relnotes.md (hunk 1 above)
Comment (jmbermingham) [reply content/en/docs/apim_relnotes/20200530_apimgr_relnotes.md:17]: Fair enough, there is no link, this is really for information, so I would say go ahead and just remove it.

File: content/en/docs/apim_relnotes/20200530_apimgr_relnotes.md (hunk 2 above)
Comment (jmbermingham) [reply content/en/docs/apim_relnotes/20200530_apimgr_relnotes.md:31]: Yes, please link to that page.

**Review by andreamussap — APPROVED** [review]
https://deploy-preview-1151--axway-open-docs.netlify.app/docs/apim_relnotes/20200530_apimgr_relnotes/#new-features-and-enhancements

**Pull request merged**